x-pack/plugins/apm/public/components/shared/PropertiesTable/index.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiIcon } from '@elastic/eui';
+import _ from 'lodash';
+import React from 'react';
+import styled from 'styled-components';
+import { colors, fontSize, px, unit, units } from '../../../style/variables';
+import { getFeatureDocs } from '../../../utils/documentation';
+// @ts-ignore
+import { ExternalLink } from '../../../utils/url';
+import { NestedKeyValueTable } from './NestedKeyValueTable';
+import PROPERTY_CONFIG from './propertyConfig.json';
+
+const indexedPropertyConfig = _.indexBy(PROPERTY_CONFIG, 'key');
+
+const TableContainer = styled.div`
+  padding-bottom: ${px(units.double)};
+`;
+
+const TableInfo = styled.div`
+  padding: ${px(unit)} 0 0;
+  text-align: center;
+  font-size: ${fontSize};
+  color: ${colors.gray2};
+  line-height: 1.5;
+`;
+
+export function getLevelOneProps(selected: string[]): string[] {
```

Comment:
What do you think about renaming this to `getTabKeys`. LevelOneProps is just such a funky name... (I know, I came up with it ;) )

Reply:
Ha yeah tbh I tried to come up with something better, failed, and just left it alone. 😆 `getPropertyTabNames` maybe? I'll go with that and we can change if that feels weird in a couple weeks. :)